### ISSUE TITLE
npm audit throwing critical warnings for out of date dependencies

### ISSUE THREAD

**Issue by ryanblock** [mhart/dynalite#90]
Hi there!

Pretty straightforward: dynalite's deps have gotten out of date, and npm's audit security report is throwing all kinds of scary (if unnecessary) warnings. Here's the current output:

```
(╯ಠ益ಠ)╯︵~/dynalite [master] npm audit

                       === npm audit security report ===

# Run  npm install --save-dev mocha@5.2.0  to resolve 2 vulnerabilities
SEMVER WARNING: Recommended action is a potentially breaking change
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ debug                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ mocha [dev]                                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ mocha > debug                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/534                       │
└───────────────┴──────────────────────────────────────────────────────────────┘


┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Critical      │ Command Injection                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ growl                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ mocha [dev]                                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ mocha > growl                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/146                       │
└───────────────┴──────────────────────────────────────────────────────────────┘


┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Memory Exposure                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ bl                                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=0.9.5 <1.0.0 || >=1.0.1                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ level-sublevel                                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ level-sublevel > levelup > bl                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/596                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 3 vulnerabilities (1 low, 1 moderate, 1 critical) in 328 scanned packages
  2 vulnerabilities require semver-major dependency updates.
  1 vulnerability requires manual review. See the full report for details.
```

Sadly, it looks like `level-sublevel` prob isn't going to be updating to remove this error any time soon ([see this issue](https://github.com/dominictarr/level-sublevel/issues/114)), but the `mocha` changes are easy enough. Will send a PR with updated deps.

**Comment by mhart** [comment]
Fixed in v1.3.0

**Comment by aassaad** [comment]
This seems to have regressed in 1.3.1
